projects/optic/src/config.ts

```diff
@@ -157,7 +157,7 @@ export async function detectCliConfig(
 ): Promise<string | undefined> {
   let currentDir = process.cwd();
   // search up from pwd to specified dir
-  while (topLevelDir.length <= currentDir.length) {
+  while (topLevelDir.length < currentDir.length) {
```

Comment:
I think we want to keep this conditional so we can check if the top level dir (e.g. `/user/some-project`) for the optic.yml

Perhaps instead we can do something like this on L173
```javascript
const next = path.dirname(currentDir)
if (next === currentDir) break
else currentDir = next
```

Reply:
cool, ill give it a try — looks like what im doing doesn't pass the tests anyway. thanks 👍